projects/js-packages/publicize-components/src/components/connection-management/connection-name.tsx

```diff
@@ -27,5 +28,5 @@ export function ConnectionName( { connection }: ConnectionNameProps ) {
 			</div>
 		);
 	}
-	return <Spinner color="black" />;
+	return <Spinner color="black" aria-label={ __( 'Loading spinner', 'jetpack' ) } />;
```

Comment:
I think this is no longer used now.

Reply:
If we still need it, then the label can be something like "Loading/Fetching account details"

Reply:
I updated this for now as you suggested, we can get rid of it when we remove the functionality from the actual code 👍